ports/psoc6/mphalport.c

```diff
@@ -63,6 +76,42 @@ int mp_hal_stdin_rx_chr(void) {
 }
 
 
+void mp_hal_pin_od_low(mp_hal_pin_obj_t pin) {
+    gpio_clear_value(pin);
+}
+
+
+void mp_hal_pin_od_high(mp_hal_pin_obj_t pin) {
+    gpio_set_value(pin);
+}
+
+
+int mp_hal_pin_read(mp_hal_pin_obj_t pin) {
+    return gpio_get_value(pin);
+}
+
+
+void mp_hal_pin_open_drain(mp_hal_pin_obj_t pin) {
+    cyhal_gpio_configure(pin, CYHAL_GPIO_DIR_INPUT,  CYHAL_GPIO_DRIVE_OPENDRAINDRIVESLOW);
+
+    // printf("2 SDA is CYHAL_GPIO_DRIVE_OPENDRAINDRIVESLOW ? %ld\n", gpio_get_drive(CYBSP_I2C_SDA));
```

Comment:
? cleanup?

Reply:
Used to test GPIO when bug was encountered. Removed it.